sidebars.yaml

```diff
@@ -33,6 +33,11 @@
                 - page: docs/use-cases/tokenization/authorized-minter.md
                 - page: docs/use-cases/tokenization/digital-artist.md
             - page: docs/use-cases/tokenization/real-world-assets.page.tsx
+            - page: docs/use-cases/tokenization/mpt-overview.md
+              expanded: false
+              items:
+                - page: docs/use-cases/tokenization/creating-an-asset-backed-multi-purpose-token.md
+                - page: docs/use-cases/tokenization/send-an-mpt.md
```

Comment:
"Send an MPT" is a very generic title, making me wonder if it should be a how-to topic instead. Maybe consider revising the title.